pokete_classes/nature.py

```diff
@@ -0,0 +1,118 @@
+"""Contains everything related to the nature of Poketes, which influences some
+of their stats"""
+
+import random
+import logging
+import scrap_engine as se
+from pokete_general_use_fns import liner
+from .ui_elements import LabelBox
+from .color import Color
+from .loops import easy_exit_loop
+
+class Nature:
+    """The Nature class itself holds basic information about the Nature,
+    that means:
+    ARGS:
+        name: The nature name
+        atc: The attack change
+        _def: The defense change
+        init: The initiative change"""
+
+    def __init__(self, name, esc="", atc=1, _def=1, init=1):
+        self.name = name
+        self.esccode = esc
+        self.atc = atc
+        self.defense = _def
+        self.initiative = init
+
+
+class PokeNature:
+    """Holds specific information about a Pokete
+    ARGS:
+        nature: The Nature of the Pokete
+        grade: The Natures grade, 1 or 2"""
+
+    natures = {name: Nature(name, **_dict) for name, _dict in {
+        "brave": {
+            "esc": Color.blue,
+            "atc": 1.1,
+            "_def": 0.9
+        },
+        "relaxed": {
+            "esc": Color.green,
+            "atc": 0.9,
+            "_def": 1.1,
+        },
+        "hasty": {
+            "esc": Color.red,
+            "_def": 0.9,
+            "init": 1.1
+        },
+        "normal": {}
```

Comment:
I would put these into `pokete_data`, as this dict does not hold logic, but data.

Putting it into `pokete_data` also means, mods can modify these values or add new natures.

Reply:
That's actually a very good point.